flask_meld/__init__.py

```diff
@@ -30,6 +30,13 @@ def init_app(self, app):
         app.jinja_env.add_extension(MeldScriptsExtension)
         app.socketio = SocketIO(app)
 
+        meld_dir = app.config.get("MELD_COMPONENT_DIR", None)
```

Comment:
This also enhances the experience for the user right? In case they want to store the components elsewhere?

Reply:
Correct